notebooks/1-lightcurves/TIKEBook_Lesson1.ipynb

```diff
@@ -0,0 +1,1265 @@
+{
```

Comment:
<p>Can you explain what "gaps cadences" means here? Do you mean "removes cadences with flagged data"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>updated the text here</p>